src/foam/nanos/crunch/services.jrl

```diff
@@ -79,6 +79,7 @@ p({
       .setAuthorize(false)
       .setInnerDAO(x.get("bareUserCapabilityJunctionDAO"))
       .setOf(foam.nanos.crunch.UserCapabilityJunction.getOwnClassInfo())
+      .setEnableInterfaceDecorators(false)
```

Comment:
So a UCJ has a few interfaces it uses
CreatedAware
LifecycleAware
...
can find them in UserCapabilityJunctionRefine.js

so if we want to turn anything off it should be just one of those... shouldn't be all with the `.setEnableInterfaceDecorators(false)`

is it possible to just remove this and run some tests. Its not clear at all why this is here (bareUser or ucj )

Reply:
The interface decorators are both on ucj and bareUcj, so they were triggered twice, and that was the reason for disabling on the higher level of the decorator chain.

Reply:
decorator chain before disabling

ProxyDAO
EasyDAO
ProxyDAO
LastModifiedByAwareDAO
LastModifiedAwareDAO
CreatedByAwareDAO
CreatedAwareDAO
RulerDAO
LifecycleAwareDAO
UserCapabilityJunctionDAO
ProxyDAO
EasyDAO
ProxyDAO
LastModifiedByAwareDAO
LastModifiedAwareDAO
CreatedByAwareDAO
CreatedAwareDAO
RulerDAO
LifecycleAwareDAO
HistoryDAO
GUIDDAO
JDAO
MDAO

Reply:
decorator chain after removing disable

ProxyDAO
EasyDAO
ProxyDAO
RulerDAO
UserCapabilityJunctionDAO
ProxyDAO
EasyDAO
ProxyDAO
LastModifiedByAwareDAO
LastModifiedAwareDAO
CreatedByAwareDAO
CreatedAwareDAO
RulerDAO
LifecycleAwareDAO
HistoryDAO
GUIDDAO
JDAO
MDAO

Reply:
interesting!